openach/tests.py

```diff
@@ -1708,3 +1708,7 @@ def test_celery_example(self):
 
         self.assertEquals(result.get(), 16)
         self.assertTrue(result.successful())
+
+    def testUpdateUrlTitleDescription(id):
```

Comment:
- Method name format should match the naming convention in the test file
- Are you going to fill this test out?